tests/llmcompressor/test_kv_cache.py

```diff
@@ -0,0 +1,312 @@
+import os
+import tempfile
+
+import pytest
+import torch
+from compressed_tensors.compressors import ModelCompressor
+from compressed_tensors.quantization.lifecycle import KVCacheScaleType
+from compressed_tensors.quantization.utils.helpers import iter_named_quantizable_modules
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from llmcompressor.core import reset_session
+from llmcompressor.transformers import oneshot
+
+MODEL_IDS = [
+    "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+    "microsoft/Phi-3-mini-4k-instruct",
```

Comment:
move to under `transformers` so that this uses the gpu runners

Reply:
ok